tests/Aspire.Dashboard.Tests/Aspire.Dashboard.Tests.csproj

```diff
@@ -6,12 +6,15 @@
 
   <ItemGroup>
     <PackageReference Include="Grpc.Tools" />
+
     <ProjectReference Include="..\..\src\Aspire.Dashboard\Aspire.Dashboard.csproj" />
 
     <Compile Include="$(TestsSharedDir)TestCertificateLoader.cs" Link="shared/TestCertificateLoader.cs" />
     <Compile Include="$(TestsSharedDir)Logging\*.cs" LinkBase="shared/Logging" />
     <Content Include="$(TestsSharedDir)TestCertificates\*.pfx" LinkBase="shared/TestCertificates" CopyToOutputDirectory="PreserveNewest" />
 
     <Content Include="..\..\src\Aspire.Dashboard\wwwroot\**\*.*" LinkBase="wwwroot" CopyToOutputDirectory="PreserveNewest" />
+
+    <ProjectReference Include="..\Aspire.EndToEnd.Tests\Aspire.EndToEnd.Tests.csproj" />
```

Comment:
If this was added for getting `PlaywrightProvider`, then instead use `<Import Project="..\Shared\Playwright\Playwright.targets" />`